README.md

```diff
@@ -75,19 +75,19 @@ Once you are ready to contribute, here the workflow you should follow:
 - Start the local web-server: `gulp` or `npm start`
 - Write some code!
 - Check to make sure your code is following style rules with `npm run stylelint`
-- Once done commit and push your changes to your fork. The linter is also run as a precommit hook.
-- Open a pull request on the origin papercss repo. Be sure to include any pictures and/or details on what you have done, it will help reviewers **a lot**!
-- When your changes are approved, they will be merged into the `develop` branch, which will finally be merged into the `master` branch when we reach a milestone in terms of features and bug fixes. Check out [Vincent Driessen's blog post](http://nvie.com/posts/a-successful-git-branching-model/), [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html), or [#27](https://github.com/rhyneav/papercss/issues/27) for more details on how this works.
+- Once done commit and push your changes to your fork. The linter is also run as a pre-commit hook.
+- Open a pull request on the origin papercss repo. Be sure to include any pictures and/or details on what you have done; it will help reviewers **a lot**!
+- When your changes are approved, they will be merged into the `develop` branch, which will finally be merged into the `master` branch when we reach a milestone regarding features and bug fixes. Check out [Vincent Driessen's blog post](http://nvie.com/posts/a-successful-git-branching-model/), [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html), or [#27](https://github.com/rhyneav/papercss/issues/27) for more details on how this works.
 
-Note: If you have a hotfix (usually typos and minor documentation tweaks), create your hotfix branch off of the master branch instead of develop: `git checkout -b hotfix-thing master`. The changes will be merged into both master and develop to keep the branches consistent.
+Note: If you have a hotfix (usually typos and minor documentation tweaks), create your hotfix branch off of the master branch instead of developing: `git checkout -b hotfix-thing master`. The changes will be merged into both the master and develop to keep the branches consistent.
```

Comment:
If I'm correct we are talking about the `develop` branch, not the term itself. Let's keep `develop` instead of `developing`.